src/main/java/uk/ac/ebi/subs/validator/filereference/FileReferenceValidatorListener.java

```diff
@@ -0,0 +1,56 @@
+package uk.ac.ebi.subs.validator.filereference;
+
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.amqp.rabbit.core.RabbitMessagingTemplate;
+import org.springframework.stereotype.Service;
+import uk.ac.ebi.subs.messaging.Exchanges;
+import uk.ac.ebi.subs.validator.data.AssayDataValidationMessageEnvelope;
+import uk.ac.ebi.subs.validator.data.SingleValidationResult;
+import uk.ac.ebi.subs.validator.data.SingleValidationResultsEnvelope;
+import uk.ac.ebi.subs.validator.data.structures.SingleValidationResultStatus;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static uk.ac.ebi.subs.validator.filereference.messaging.FileReferenceQueues.FILE_REFERENCE_ASSAYDATA_VALIDATION;
+import static uk.ac.ebi.subs.validator.filereference.messaging.FileReferenceRoutingKeys.EVENT_VALIDATION_ERROR;
+import static uk.ac.ebi.subs.validator.filereference.messaging.FileReferenceRoutingKeys.EVENT_VALIDATION_SUCCESS;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class FileReferenceValidatorListener {
```

Comment:
I've just realised that in every validator we call the listener `ValidatorListener` and since it's under _filereference_ we know to which validator it refers. For the sake of consistency we should rename it.